Fix & imporve links

alehandrof opened the pull request.

jackmcdade commented:
Technically Pages, Collection, and Taxonomy are also Relate fieldtypes, but maybe that doesn't matter. Clarity is a good thing. Thanks!

alehandrof commented:
Ah, I see what you mean. I just fixed the broken link that was already there and (since I was editing anyway), added a link one for the get modifier. What you're suggesting would require more robust edits than I'm confident in doing right now. I'm still just finding my footing with v2.

jackmcdade commented:
No worries, noted!
